content/english/security/3-activity/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Activity"
+description: ""
+draft: true
+weight: 3
+translationKey: "security-3"
+---
+
+
+
+
+Let's play a game!
+
+
+
+<form accept-charset="UTF-8" action="https://formkeep.com/f/exampletoken" method="POST">
+  <input type="hidden" name="utf8" value="✓">
+  Write your favorite sentence in the text box below. Make sure it has atleast five words. (E.g. I like playing volleyball at the beach.)
+  <input type="email"  name="email" placeholder="Your Favorite sentence.">
+</br>
+   Now take the first letter of each word from the above sentence and write inside the text box. (E.g. ilpvatb)
+  <input type="text" name="name" placeholder="The first letter of each word">
+</br>
+   Add some numbers to the text. Remember the position of the numbers. (E.g. 1ilpvatb6)
+  <input type="tel" name="phone" placeholder="Add some numbers">
+</br>
+  Finally, let's add some special charcter to the text. (E.g. 1ilpv$atb6)
+  <input type="tel" name="phone" placeholder="Add a special character">
+
+</br> <!--<button type="submit">Submit</button>-->
+</form>
+
+
+### Voila that's your new password! Doesn't it look hard to crack?
+
+
+<!---Open your favorite internet browser (such as Edge, Firefox, Google Chrome, Internet Explorer), and go to https://aka.ms/oranges. Your screen should look like this:
```

Comment:
Do you still need these Adafruit related work?

Reply:
nope, i went ahead and removed it